sdk/src/claim.rs

```diff
@@ -2003,12 +2007,13 @@ impl Claim {
             // for v1 claims check the single assertion store
             if let Some(assertion) = all_actions.first() {
                 let first_actions = Actions::from_assertion(assertion.assertion())?;
-                let first_actions_first_action = &first_actions.actions()[0];
-
-                if first_actions_first_action.action() == c2pa_action::OPENED
-                    || first_actions_first_action.action() == c2pa_action::CREATED
-                {
-                    found_first_action = Some(assertion);
+                let first_actions_first_action = &first_actions.actions().first();
```

Comment:
For v1 claims, this rule does not apply and we should not try to enforce it here.